manifest-build-tools/application/reprove.py

```diff
@@ -243,15 +426,38 @@ def main():
 
     if args.git_credential:
         manifest_actions.set_git_credentials(args.git_credential)
-
+    
```

Comment:
Because there are optional parameters.
These properties are set a default value in init().
If these parameters are provided, we can set the value of them out of init().
That's the reason for creating function setter.